app/src/main/java/com/example/campusguide/Bootstrapper.kt

```diff
@@ -28,7 +29,8 @@ class Bootstrapper constructor(activity: MapsActivity) {
         val permissions = Permissions(activity)
 
         // Search
-        val search = Search(activity, map)
+        //val search = Search(activity, map)
```

Comment:
Same for this as it goes with the import

Reply:
Removed